DoNotCallDangerousMethodsInDeserialization analyzer fails CoreClr build

#### Analyzer package

[microsoft.netcore.analyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)

#### Package Version

Current repo build

#### Diagnostic ID

CA5360 DoNotCallDangerousMethodsInDeserialization 

#### Repro steps

1. build roslyn-analyzers with runtime repo version (3.0.0)
2.  Copy *.dll files from `[repo]\artifacts\bin\Microsoft.NetCore.CSharp.Analyzers\Debug\netstandard2.0` into your nugget packages corresponding folder ( for example: `C:\Users\username\.nuget\packages\microsoft.netcore.analyzers\3.0.0-beta4.20177.1\analyzers\dotnet\cs`
3. build runtime repo (full build or coreclr only)

#### Expected behavior
Build succeeds
#### Actual behavior
Build fails, strack trace:
```
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error : Process terminated. Assertion failed. [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error : AllocateNewUninitializedArray was not found in callGraph [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>g__FindCalledDangerousMethod|3(ISymbol methodSymbol, HashSet`1 visited, Dictionary`2 results) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.NetCore.Analyzers.Security.DoNotCallDangerousMethodsInDeserialization.<>c__DisplayClass8_0.<Initialize>b__2(CompilationAnalysisContext compilationAnalysisContext) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.<>c.<ExecuteCompilationActionsCore>b__46_1(ValueTuple`2 data) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.ExecuteAndCatchIfThrows_NoLock[TArg](DiagnosticAnalyzer analyzer, Action`1 analyze, TArg argument, Nullable`1 info) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.ExecuteAndCatchIfThrows[TArg](DiagnosticAnalyzer analyzer, Action`1 analyze, TArg argument, Nullable`1 info) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.ExecuteCompilationActionsCore(ImmutableArray`1 compilationActions, DiagnosticAnalyzer analyzer, AnalyzerStateData analyzerStateOpt) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.TryExecuteCompilationActions(ImmutableArray`1 compilationActions, DiagnosticAnalyzer analyzer, CompilationEvent compilationEvent, AnalysisScope analysisScope, AnalysisState analysisStateOpt) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.TryExecuteCompilationActions(ImmutableDictionary`2 compilationActionsMap, CompilationEvent compilationEvent, AnalysisScope analysisScope, AnalysisState analysisStateOpt, CancellationToken cancellationToken) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.TryProcessEventCoreAsync(CompilationEvent e, AnalysisScope analysisScope, AnalysisState analysisStateOpt, CancellationToken cancellationToken) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.TryProcessEventCoreAsync(CompilationEvent e, AnalysisScope analysisScope, AnalysisState analysisStateOpt, CancellationToken cancellationToken) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.ProcessEventAsync(CompilationEvent e, AnalysisScope analysisScope, AnalysisState analysisStateOpt, CancellationToken cancellationToken) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.ProcessEventAsync(CompilationEvent e, AnalysisScope analysisScope, AnalysisState analysisStateOpt, CancellationToken cancellationToken) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.ProcessCompilationEventsAsync(AnalysisScope analysisScope, AnalysisState analysisStateOpt, Boolean prePopulatedEventQueue, CancellationToken cancellationToken) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.ExecutionContextCallback(Object s) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext(Thread threadPoolThread) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext() [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction(IAsyncStateMachineBox box, Boolean allowInlining) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunContinuations(Object continuationObject) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.TrySetResult() [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.WhenAllPromise.Invoke(Task completedTask) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunOrQueueCompletionAction(ITaskCompletionAction completionAction, Boolean allowInlining) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunContinuations(Object continuationObject) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task`1.TrySetResult(TResult result) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.UnwrapPromise`1.TrySetFromTask(Task task, Boolean lookForOce) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.UnwrapPromise`1.Invoke(Task completingTask) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunOrQueueCompletionAction(ITaskCompletionAction completionAction, Boolean allowInlining) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunContinuations(Object continuationObject) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task`1.TrySetResult(TResult result) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.SetResult(TResult result) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.<>c__DisplayClass102_0.<<ProcessCompilationEventsAsync>b__1>d.MoveNext() [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.ExecutionContextCallback(Object s) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext(Thread threadPoolThread) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext() [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction(IAsyncStateMachineBox box, Boolean allowInlining) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunContinuations(Object continuationObject) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task`1.TrySetResult(TResult result) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.ProcessCompilationEventsCoreAsync(AnalysisScope analysisScope, AnalysisState analysisStateOpt, Boolean prePopulatedEventQueue, CancellationToken cancellationToken) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.ExecutionContextCallback(Object s) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext(Thread threadPoolThread) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext() [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction(IAsyncStateMachineBox box, Boolean allowInlining) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunContinuations(Object continuationObject) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task`1.TrySetResult(TResult result) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.ProcessEventAsync(CompilationEvent e, AnalysisScope analysisScope, AnalysisState analysisStateOpt, CancellationToken cancellationToken) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.ExecutionContextCallback(Object s) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext(Thread threadPoolThread) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext() [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction(IAsyncStateMachineBox box, Boolean allowInlining) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunContinuations(Object continuationObject) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task`1.TrySetResult(TResult result) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncValueTaskMethodBuilder`1.SetResult(TResult result) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.TryProcessEventCoreAsync(CompilationEvent e, AnalysisScope analysisScope, AnalysisState analysisStateOpt, CancellationToken cancellationToken) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.ExecutionContextCallback(Object s) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext(Thread threadPoolThread) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext() [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction(IAsyncStateMachineBox box, Boolean allowInlining) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunContinuations(Object continuationObject) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task`1.TrySetResult(TResult result) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncValueTaskMethodBuilder`1.SetResult(TResult result) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.TryProcessSymbolDeclaredAsync(SymbolDeclaredCompilationEvent symbolEvent, AnalysisScope analysisScope, AnalysisState analysisStateOpt, CancellationToken cancellationToken) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.ExecutionContextCallback(Object s) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext(Thread threadPoolThread) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext() [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction(IAsyncStateMachineBox box, Boolean allowInlining) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunContinuations(Object continuationObject) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.GetPerSymbolAnalyzerActionsAsync(ISymbol symbol, AnalysisScope analysisScope, AnalysisState analysisStateOpt, CancellationToken cancellationToken) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.ExecutionContextCallback(Object s) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext(Thread threadPoolThread) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext() [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction(IAsyncStateMachineBox box, Boolean allowInlining) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunContinuations(Object continuationObject) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.GetPerSymbolAnalyzerActionsAsync(ISymbol symbol, DiagnosticAnalyzer analyzer, AnalysisState analysisStateOpt, CancellationToken cancellationToken) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.ExecutionContextCallback(Object s) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext(Thread threadPoolThread) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext() [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction(IAsyncStateMachineBox box, Boolean allowInlining) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunContinuations(Object continuationObject) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.<GetPerSymbolAnalyzerActionsAsync>g__getSymbolActionsCoreAsync|123_1(AnalyzerDriver driver, ISymbol symbol, DiagnosticAnalyzer analyzer) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.ExecutionContextCallback(Object s) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext(Thread threadPoolThread) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext() [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction(IAsyncStateMachineBox box, Boolean allowInlining) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunContinuations(Object continuationObject) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerManager.GetPerSymbolAnalyzerActionsAsync(ISymbol symbol, DiagnosticAnalyzer analyzer, AnalyzerExecutor analyzerExecutor) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.ExecutionContextCallback(Object s) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext(Thread threadPoolThread) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext() [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction(IAsyncStateMachineBox box, Boolean allowInlining) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunContinuations(Object continuationObject) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task`1.TrySetResult(TResult result) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.SetResult(TResult result) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerManager.GetSymbolAnalysisScopeAsync(ISymbol symbol, DiagnosticAnalyzer analyzer, ImmutableArray`1 symbolStartActions, AnalyzerExecutor analyzerExecutor) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.ExecutionContextCallback(Object s) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext(Thread threadPoolThread) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext() [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction(IAsyncStateMachineBox box, Boolean allowInlining) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunContinuations(Object continuationObject) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task`1.TrySetResult(TResult result) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.SetResult(TResult result) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at Microsoft.CodeAnalysis.Diagnostics.AnalyzerManager.GetSymbolAnalysisScopeCoreAsync(ISymbol symbol, ImmutableArray`1 symbolStartActions, AnalyzerExecutor analyzerExecutor, AnalyzerExecutionContext analyzerExecutionContext) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.ExecutionContextCallback(Object s) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext(Thread threadPoolThread) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.AsyncStateMachineBox`1.MoveNext() [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.AwaitTaskContinuation.RunOrScheduleAction(IAsyncStateMachineBox box, Boolean allowInlining) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.RunContinuations(Object continuationObject) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.FinishSlow(Boolean userDelegateExecute) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread) [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Users\buyaa\.nuget\packages\microsoft.net.compilers.toolset\3.6.0-2.20166.2\tasks\netcoreapp3.1\Microsoft.CSharp.Core.targets(59,5): error :    at System.Threading.ThreadPoolWorkQueue.Dispatch() [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]
```

@mavasani I don't see any other warning for any other analyzer except one from CA1830: `D:\dotnet\runtime\src\libraries\System.Private.CoreLib\src\System\Text\StringBuilder.cs(1147,59): error CA1830: Remove the ToString call in order to use a strongly-typed StringBuilder overload. [D:\dotnet\runtime\src\coreclr\src\System.Private.CoreLib\System.Private.CoreLib.csproj]` 
CC @stephentoub 

For false positives only checked that for ArgumentException analyzer

--- a/src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Security/DoNotCallDangerousMethodsInDeserialization.cs
+++ b/src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Security/DoNotCallDangerousMethodsInDeserialization.cs
@@ -180,8 +180,12 @@ namespace Microsoft.NetCore.Analyzers.Security
 
                                 calledMethods.TryAdd(calledSymbol, true);
 
+                                // calledSymbol.ContainingSymbol.Kind == SymbolKind.Method => local function
+                                // For the purposes of this rule, we'll treat invocations inside the local function as part of
+                                // the containing method's set of invocations.
                                 if (!calledSymbol.IsInSource() ||
                                     calledSymbol.ContainingType.TypeKind == TypeKind.Interface ||
+                                    calledSymbol.ContainingSymbol.Kind == SymbolKind.Method ||
                                     calledSymbol.IsAbstract ||
                                     possibleDelegateSymbol.TypeKind == TypeKind.Delegate)
                                 {

--- a/src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Security/DoNotCallDangerousMethodsInDeserializationTests.cs
+++ b/src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Security/DoNotCallDangerousMethodsInDeserializationTests.cs
@@ -2499,6 +2499,59 @@ public class TestClass : IDisposable
 }");
         }
 
+        [Fact]
+        public async Task TestLocalFunctionDiagnostic()
+        {
+            await VerifyCS.VerifyAnalyzerAsync(@"
+using System;
+using System.IO;
+using System.Runtime.Serialization;
+
+[Serializable()]
+public class TestClass
+{
+    private string member;
+
+    [OnDeserializing()]
+    internal void OnDeserializingMethod(StreamingContext context)
+    {
+        byte[] bytes = new byte[] {0x20, 0x20, 0x20};
+        ALocalFunction();
+
+        void ALocalFunction()
+        {
+            File.WriteAllBytes(""C:\\"", bytes);
+        }
+    }
+}",
+            GetCSharpResultAt(12, 19, "TestClass", "OnDeserializingMethod", "WriteAllBytes"));
+        }
+
+        [Fact]
+        public async Task TestLocalFunctionNoDiagnostic()
+        {
+            await VerifyCS.VerifyAnalyzerAsync(@"
+using System;
+using System.Runtime.Serialization;
+
+[Serializable()]
+public class TestClass
+{
+    private string member;
+
+    [OnDeserializing()]
+    internal void OnDeserializingMethod(StreamingContext context)
+    {
+        ALocalFunction();
+
+        void ALocalFunction()
+        {
+            object o = new Object();
+        }
+    }
+}");
+        }
+
         private static DiagnosticResult GetCSharpResultAt(int line, int column, params string[] arguments)
             => VerifyCS.Diagnostic()
                 .WithLocation(line, column)

